ioctl/util/util.go

```diff
@@ -221,3 +222,15 @@ func ParseHdwPath(addressOrAlias string) (uint32, uint32, uint32, error) {
 func AliasIsHdwalletKey(addressOrAlias string) bool {
 	return strings.HasPrefix(strings.ToLower(addressOrAlias), "hdw::")
 }
+
+// IsNewfoundland returns whether the height passed NewfoundlandHeight
+func IsNewfoundland(chainID uint32, h uint64) bool {
+	var newfoundland uint64 = math.MaxUint64
+	switch chainID {
+	case 1:
+		newfoundland = 17027641
+	case 2:
+		newfoundland = 14057281
+	}
+	return h >= newfoundland
+}
```

Comment:
not sure about this implementation, which includes hard-coded heights

Reply:
just some filler number for now
when we release the 1.8.0 HF, needs to update here accordingly, added a TODO